src/encoded/region_search.py

```diff
@@ -402,20 +402,27 @@ def regulome_summary(context, request):
         # Get rsid for the coordinate
         rsids = get_rsids(atlas, assembly, chrom, start, end)
         # Only SNP or single nucleotide are considered as scorable
+        features = dict()
+        regulome_score = 'N/A'
         if rsids == [] and (int(end) - int(start)) > 1:
-            regulome_score = 'N/A'
             result['notifications'].append({coord: 'Failed: {}'.format(
                 'Non-SNP or multi-nucleotide region is not scorable')})
         else:  # Scorable
             try:
                 all_hits = region_get_hits(atlas, assembly, chrom, start, end)
-                regulome_score = atlas.regulome_score(all_hits['datasets'])
+                evidence = atlas.regulome_evidence(all_hits['datasets'])
```

Comment:
I feel like regulome_summary and regulome_search have more than a little redundant code.   Discussed with @emmao making the summary do SNP fetches in the background asynchronous which might make most of this view code irrelvant.